lua/utils.lua

```diff
@@ -0,0 +1,60 @@
+function M.tbl_length(T)
```

Comment:
🚓 Unfortunately fzf-lua is licensed under AGPL, we can't just copy/paste it. :( I'd love to hear more about the issues you were having though, and possibly look for another solution.

Reply:
no idea how exactly copyright works, but we can rename the variables, I don't think he has trademark for the word get_visual_selection or use of the word csrow or any nvim api used in this way or if conditions with vim mode combination... That would be absolutely insane if that's true, most open source would be unusable. I mean every time you write some snippet of code or if vim.fn.mod() you'd need to double check if there is a project with the same code. I mean you would quickly run out of code you can right using open apis, that is not copyrighted no ?

Reply:
I'm no expert either, so I tend to be cautious. It's why I find myself limiting to projects with appropriate licensing for anything I work on.